assets/wizards/dashboard/views/dashboardCard.js

```diff
@@ -26,21 +41,33 @@ class DashboardCard extends Component {
 	 * Render.
 	 */
 	render() {
-		const { name, description, slug, url, svg, status } = this.props;
+		const { name, description, slug, url, status } = this.props;
 		const classes = classNames( 'newspack-dashboard-card', slug, status );
-		const iconLink = (
-			<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-				<Path d="M10.02 6L8.61 7.41 13.19 12l-4.58 4.59L10.02 18l6-6-6-6z" />
-			</SVG>
-		);
-		const iconDone = (
-			<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" className="icon-completed">
-				<Path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z" />
-			</SVG>
-		);
+		let icon;
```

Comment:
Suggestion: make the logic a bit lighter by defining an object that maps icons to  the slugs:

```
const iconMap = {
	'site-design': <WebIcon />,
  'reader-revenue': <AccountBalanceWalletIcon />,
  advertising: <FeaturedVideoIcon />,
  syndication: <SyncAltIcon />,
  analytics: <TrendingUpIcon />,
  performance: <SpeedIcon />,
  seo: <SearchIcon />,
  'health-check': <HealingIcon />,
  engagement: <ForumIcon />,
};
```

Then the line to render the icon reduces to just this:

```
{ iconMap[ slug ] || <WidgetsIcon /> }
```

Reply:
Thank you for the suggestion. This is much better.
Fixed in https://github.com/Automattic/newspack-plugin/pull/356/commits/1b88541aeb31cbc1fa404175b4cc20fca9c4feb5